proto/expression.proto

```diff
@@ -222,6 +222,11 @@ enum ScalarFuncSig {
 	ModDecimal = 216;
 	ModInt = 217;
 	MultiplyIntUnsigned = 218;
+	PlusIntUnsignedUnsigned  = 219;
+	PlusIntUnsignedSigned   = 220;
+	PlusIntSignedUnsigned   = 221;
+	PlusIntSignedSigned     = 222;
```

Comment:
Prefer to be `PlusIntSignedSigned = 222;` instead to be aligned with styles above.